perception_eval/test/perception_lsim2d.py

```diff
@@ -64,15 +64,21 @@ def __init__(
                 else None,
             )
         )
+        evaluation_config_dict.update(
+            dict(
+                allow_matching_unknown=True,
+                merge_similar_labels=False,
+                label_prefix="autoware",
```

Comment:
lable_prefix must be variable. not literal

```suggestion
                label_prefix=self.label_prefix,
```